test/clj/rems/test_db.clj

```diff
@@ -65,52 +63,6 @@
   (is (= #{:logged-in} (roles/get-roles "juho"))) ;; default role
   (is (thrown? RuntimeException (roles/add-role! "pekka" :unknown-role))))
 
-(deftest test-get-entitlements-for-export
```

Comment:
So is there any test for getting actual CSV content out?

Reply:
yes, `rems.api.test-entitlements` namespace contains API tests that fetch entitlements from the `/csv` route. the `test-db` namespace is redundant, if we can also test the API and different roles. i can test the other scenarios here as well, once i get them implemented in application code.